src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -75,6 +81,14 @@ public Locomotive(ConductorConfig configuration, AppiumDriver driver) {
         init(configuration, driver);
     }
 
+    public AppiumDriver getDriver() {
+        return driver.get();
+    }
+
+    public void setDriver(AppiumDriver d) {
```

Comment:
minor suggestion is to not use abbreviation in variable names: `driver` instead of `d`